### ISSUE TITLE
User can purchase a lesson

### ISSUE THREAD

**Issue by canopytrading** [canopytradinginc/Roadmap#2]
- [ ] User can view a lesson from the content roadmap page (name tbd)
- [ ] User can click on lesson to read a short description of the lesson
- [ ] User can view a profile or short bio of a trader when clicking on their content feed 
- [ ] User can add the lesson or feed to their cart 
- [ ] User can checkout 
- [ ] User can view their purchase (and others cannot)

**Comment by kevin1chun** [comment]
@b3ckm4n
Is this post-squarespace?

**Comment by b3ckm4n** [comment]
Nope, this is on Squarespace. I don't think we need to code any of it, but I figured we could use this when we test to make sure we got our bases covered.
